css-layout-classes/index.html

```diff
@@ -0,0 +1,94 @@
+</head>
+<body>
+<div class="container">
+  <div class="row">
+    <h1>Choose your character!</h1>
+  </div>
+  <div class="row">
+
+    <div class="column-one-third">
+      <img src="images/link.png">
+    </div>
+
+    <div class="column-one-third">
+      <h2>Super Smash Bros. Ultimate</h2> <!--Fonts have been set to san serif like example.png appears to have-->
+      <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+      dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+      book. It has survived not only five centuries, but also the leap into elec.</p>
+    </div>
+
+    <div class="column-one-third">
+      <img src="images/yoshi.png" />
+    </div>
+    </div>
+
+
+
+    <div class="row">
+      <div class="column-one-fourth">
+        <img src="images/kirby.png" />
+      </div>
+
+      <div class="column-one-fourth">
+        <img src="images/fox.png" />
+      </div>
+
+      <div class="column-one-fourth">
+          <img src="images/pikachu.png" />
+      </div>
+
+      <div class="column-one-fourth">
+          <img src="images/ness.png" />
+      </div>
+    </div>
+
+
+    <div class="row">
+      <div class="column-two two-thirds">
+        <h2>Donkey Kong</h2>
+        <!--This text and the mario text have been set to be the same number of lines as in example.png-->
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard
+        dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen
+        book. It has survived not only five centuries. </p>
+        <p> It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more
+        recently with. It has survived not only  unchanged.</p>
+      </div>
+
+      <div class="column-two one-third"  >
+        <img src="images/dk.png" />
+      </div>
+    </div>
+
+
+
+
+    <div class="row">
+      <div class="column-two one-third" >
+        <img src="images/mario.png" />
+      </div>
+
+      <div class="column-two two-thirds"  >
+        <h2>Mario</h2>
+        <p>Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's
+          standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type
+          specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining
+          essentially unchanged.</p>
+        <p> It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more
+          recently with. It has survived not only five centuries, but also the leap into electronic typesetting.</p>
+      </div>
+    </div>
+
+  </div>
```

Comment:
Your indentation here is still incorrect, please revise it. It may seem like I'm being nitpicky, but I promise it will be worthwhile to format your code correctly. Also, you can remove any empty newlines between elements, HTML typically doesn't have empty lines between elements.